src/icon-manifest/supportedExtensions.ts

```diff
@@ -12,6 +12,7 @@ export const extensions: IFileCollection = {
     { icon: 'ai', extensions: ['ai'], format: FileFormat.svg },
     { icon: 'ai2', extensions: ['ai'], format: FileFormat.svg, disabled: true },
     { icon: 'angular', extensions: ['.angular-cli.json', 'angular-cli.json'], filename: true, format: FileFormat.svg },
+    { icon: 'anyscript', languages: [languages.anyscript], format: FileFormat.svg },
```

Comment:
Move it after `ansible` to keep alphabetic ordering. Don't let `ng` mislead you they are there because they are `angular` related.